static/app/views/projectInstall/createProject.tsx

```diff
@@ -481,6 +481,7 @@ export function CreateProject() {
           </HelpText>
           <StyledListItem>{t('Choose your platform')}</StyledListItem>
           <PlatformPicker
+            key={formData.platform?.category}
```

Comment:
we need this here because until the data is returned from the localStorage this can be `undefined` and without the `useEffect` in the PlatformPicker this never gets re rendered.

Reply:
I am not happy with this, but don't want to introduce a loading state or something else here now.